### DISCUSSION TITLE
Copilt in neovim error

### DISCUSSION THREAD

**Discussion by Eloco** [community/community#16890] [Copilot Conversations] [answered]
recently i suddenly cant use copilot in neovim  properly.
after i upgrade the copilot, it still got bug.

NVIM v0.8.0-dev

```
:Copilot restart
:Copilot log
agent stopped
agent exited with status 0
```

How can i got the detail msg for the bug?

**Comment by tpope** [comment, marked as answer]
There's currently an issue running on Node.js 18.1.0+. Downgrading to an older version is a workaround.  If you can't easily downgrade, you can use a tool like [nodenv](https://github.com/nodenv/nodenv) or [nvm](https://github.com/nvm-sh/nvm) to install a version of Node in your home directory, and then tell Copilot where to find it:

```vim
 let g:copilot_node_command = "~/.nodenv/versions/16.15.0/bin/node"
```

If changing Node.js versions doesn't solve your problem, please report back.

**Reply by Eloco** [reply]
emm, my node version is not so high.

v14.17.0

**Reply by tpope** [reply]
Apple Silicon? Node.js older than version 16 likes to crash there.
